Mods/Alpakit/Source/Alpakit/Public/AlpakitSettings.h

```diff
@@ -49,13 +49,13 @@ class ALPAKIT_API UAlpakitSettings : public UObject {
     /** Saves alpakit settings to configuration file */
     void SaveSettings();
 
-    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Windows")
+    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Windows", meta = (ToolTip = "After packaging settings for the Windows target"))
     FAlpakitTargetSettings WindowsGameTargetSettings;
 
-    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Windows Server")
+    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Windows Server", meta = (ToolTip = "After packaging settings for the Windows Server target"))
     FAlpakitTargetSettings WindowsServerTargetSettings;
 
-    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Linux Server")
+    UPROPERTY(EditAnywhere, config, Category = Config, DisplayName="Linux Server", meta = (ToolTip = "After packaging settings for the Linux Server target"))
```

Comment:
Same thing as above, not after packaging settings, but packaging settings as a whole for dev mode